add clear error message when Ruby is missing

As someone wanting to contribute some pages, this was confusing:

    me@laptop: ~/code/tldr$ make setup
    Git pre-commit hook installed.
    make: bundle: Command not found
    Makefile:21: recipe for target 'deps' failed
    make: *** [deps] Error 127

In #428 it appears the solution (on Debian) is:

    sudo apt-get install ruby
    gem install bundler

Make setup should say something more clear about what the issue is.

--- a/Makefile
+++ b/Makefile
@@ -1,7 +1,5 @@
 default: lint
 
-all: setup index
-
 index:
 	@echo "WARNING!"
 	@echo "Index rebuilding is deprecated."
@@ -10,7 +8,18 @@ index:
 	@TLDRHOME=`pwd` ./scripts/build_index.rb
 	@echo "Index rebuilt."
 
-setup: hooks deps
+setup: prerequisites hooks deps
+
+prerequisites:
+	@echo
+	@echo "IMPORTANT!"
+	@echo "Before setting up hooks, make sure you have read Contributing Guidelines"
+	@echo "https://github.com/tldr-pages/tldr/blob/master/CONTRIBUTING.md#submitting-a-pull-request"
+	@echo
+	@echo "TL;DR:"
+	@echo "1. Install Ruby suitable for your system"
+	@echo "2. Run 'gem install bundler'"
+	@echo
 
 hooks:
 	@cp ./scripts/pre-commit .git/hooks
@@ -24,4 +33,4 @@ deps:
 lint:
 	@bundle exec mdl --style ./scripts/markdown-style.rb pages
 
-.PHONY: default index setup hooks deps lint
+.PHONY: default index setup prerequisites hooks deps lint

